### PR TITLE
Added a test for XXXMBE in TUNING

### PR DESCRIPTION
**Pull request by vkip**
To be used with code and tests added in https://github.com/OPM/opm-simulators/pull/4621

### CODE REVIEW

**Review by bska — COMMENTED** [review]
Looks good from a quick glance.  We might want to scrub keywords that are not supported by OPM Flow, though.  Similarly for the parts of the `SCHEDULE` section that are disabled/commented out, unless those parts will be re-enabled at a later time.

File: tuning/01_TUNING_XXXMBE.DATA [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+RUNSPEC 
+-- ========================================================================
+
+
+TITLE
+   BASE MODEL 1
+
+ 
+-- Turns off echoing of all the input files to the print file.  
+NOECHO
```
[the remark this answers is not in the document]
Comment (vkip) [reply tuning/01_TUNING_XXXMBE.DATA]: Good points, cleaned and removed NOECHO.

**Review by bska — COMMENTED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into the master branch when you're ready.

**Comment by vkip** [comment]
> Thanks a lot for the updates. This looks good to me now and I'll merge into the master branch when you're ready.

Great, thanks. I'm ready :)

**Review by bska — APPROVED** [review]
>>Thanks a lot for the updates. This looks good to me now and I'll merge into the master branch when you're ready.
>
>Great, thanks. I'm ready :)

Excellent.  I'll merge into master, then.

**Pull request merged**